aamt_parse2.pm

```diff
@@ -941,7 +1212,7 @@ sub pars_GetFileFromSource
 sub pars_Generate2D
 {   
     utf_setCurrentModuleName('PARSER');
-    ilog_print(1,"\n\nParsing Apache conf file(s)... ");
+#    ilog_print(1,"\n\nParsing Apache conf file(s)... ");
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can skip this, if it is false, then keep the current code.